{"name": "邮箱重复时返回错误信息", "status": "passed", "attachments": [{"name": "log", "source": "7f455ead-c93e-4aa3-9527-47a6fa0c6bc1-attachment.txt", "type": "text/plain"}], "start": 1741179650657, "stop": 1741179652145, "uuid": "926be8e7-e718-4d7f-b9d7-12380ef93e43", "historyId": "2cb599d31c902407a349da57407b7973", "testCaseId": "e9e7c8aee46571bad0866d69dd275e54", "fullName": "test_case.test_customer.TestCustomer#test_update_customer_email", "labels": [{"name": "feature", "value": "用户中心"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_customer"}, {"name": "subSuite", "value": "TestCustomer"}, {"name": "host", "value": "bindeMacBook-Pro.local"}, {"name": "thread", "value": "10106-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_customer"}]}